### ISSUE TITLE
No Explorer data, No Log View etc. - only config&InfluxDB Admin working

### ISSUE THREAD

**Issue by nickma82** [influxdata/chronograf#5142]
Hi, I'm running InfluxDB 1.7.4 and Chronograf 1.7.9 inside VirtualBox (guest and host ubuntu 18.04.2).
I forwarded the following ports to the host and are accessing chronograf from the host:
```
  # Snapshot from my Vagrantfile
  config.vm.network "forwarded_port", guest:8086, host: 8086
  config.vm.network "forwarded_port", guest:8888, host: 8888
```

Now, I'm able to access localhost:8888 - the chronograf GUI and I set it up (without a Kapacitor) and it's giving me correct information (about which table is available) in the "InfluxDB Admin" pane - so I assume the connection is correct.
BUT I'm not able to do any queries nor see logs or other content related stuff of the databases.

Query from influx directly from inside the guest - same query 
```
vagrant@ubuntu-bionic:~$ influx -username root -password xxxxxx
Connected to http://localhost:8086 version 1.7.4
InfluxDB shell version: 1.7.4
Enter an InfluxQL query
> use telegraf
Using database telegraf
> select * from weather
name: weather
time                host          temp topic
----                ----          ---- -----
1554318034468085132 ubuntu-bionic 42   telegraf/test
```

Log of chronograf:
```
vagrant@ubuntu-bionic:~$ chronograf --log-level=debug --reporting-disabled
...
DEBU[0031] Request                                       component=server method=GET remote_addr=10.0.2.2:41088 url=/chronograf/v1/sources/1/dbs
DEBU[0031] query                                         command=SHOW DATABASES component=proxy db= host=localhost:8086 rp=
INFO[0031] Response: No Content                          component=server method=GET remote_addr=10.0.2.2:41086 response_time=1.920293ms status=204
DEBU[0031] Request                                       component=server method=GET remote_addr=10.0.2.2:41094 url=/chronograf/v1/sources/1/users
DEBU[0031] query                                         command=SHOW USERS component=proxy db= host=localhost:8086 rp=
DEBU[0031] query                                         command=SHOW RETENTION POLICIES ON "telegraf" component=proxy db=telegraf host=localhost:8086 rp=
DEBU[0031] query                                         command=SHOW GRANTS FOR "root" component=proxy db= host=localhost:8086 rp=
DEBU[0031] query                                         command=SHOW RETENTION POLICIES ON "_internal" component=proxy db=_internal host=localhost:8086 rp=
INFO[0031] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41094 response_time=6.566446ms status=200
INFO[0031] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41088 response_time=11.725646ms status=200
DEBU[0031] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/sources/1/permissions
INFO[0031] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41086 response_time=1.01275ms status=200
DEBU[0031] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/sources/1/dbs
DEBU[0031] query                                         command=SHOW DATABASES component=proxy db= host=localhost:8086 rp=
DEBU[0031] query                                         command=SHOW RETENTION POLICIES ON "telegraf" component=proxy db=telegraf host=localhost:8086 rp=
DEBU[0031] query                                         command=SHOW RETENTION POLICIES ON "_internal" component=proxy db=_internal host=localhost:8086 rp=
INFO[0031] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41086 response_time=4.849183ms status=200
DEBU[0033] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/me
INFO[0033] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41086 response_time=117.343µs status=200
DEBU[0035] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/sources/1/health
INFO[0035] Response: No Content                          component=server method=GET remote_addr=10.0.2.2:41086 response_time=1.633674ms status=204
DEBU[0035] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/laser-spinner.0560a4d9.png
INFO[0035] Serving assets                                component=server method=GET remote_addr=10.0.2.2:41086 url=/laser-spinner.0560a4d9.png
INFO[0035] Response: Not Modified                        component=server method=GET remote_addr=10.0.2.2:41086 response_time=1.689517ms status=304
DEBU[0035] Request                                       component=server method=POST remote_addr=10.0.2.2:41088 url=/chronograf/v1/sources/1/proxy
DEBU[0035] Request                                       component=server method=POST remote_addr=10.0.2.2:41086 url=/chronograf/v1/sources/1/proxy
DEBU[0035] query                                         command=SHOW DATABASES component=proxy db= host=localhost:8086 rp=
DEBU[0035] query                                         command=select * from weather component=proxy db= host=localhost:8086 rp=
INFO[0035] Response: OK                                  component=server method=POST remote_addr=10.0.2.2:41088 response_time=2.230781ms status=200
INFO[0035] Response: OK                                  component=server method=POST remote_addr=10.0.2.2:41086 response_time=2.749895ms status=200
DEBU[0044] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/me
INFO[0044] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41086 response_time=242.859µs status=200
DEBU[0054] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/me
INFO[0054] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41086 response_time=88.855µs status=200
DEBU[0064] Request                                       component=server method=GET remote_addr=10.0.2.2:41086 url=/chronograf/v1/me
INFO[0064] Response: OK                                  component=server method=GET remote_addr=10.0.2.2:41086 response_time=98.72µs status=200
....
```

Any ideas?
Best

**Comment by endresjo** [comment]
Same problem here; 
1. Added influxdb source (tried https and http) in Chronograf 
2. In Chronograf: 
   - Shows correct info under "Configuration" panel
   - Shows correct info under "InfluxDB Admin" panel (all databases listed; databases can be modified)
   - Under "Host List" and "Explore" panel no entries; debug showing empty query response for commands
3. Tested with 
    - influxdb installed in both Vagrant and on VMware Ubuntu VM
    - chronograf installed on same VM as well as docker container; versions 1.7 and above show this bug; chronograf version 1.62 working fine with same influxdb database

> InfluxDB v1.7.6 (git: 1.7 01c8dd416270f424ab0c40f9291e269ac6921964)
> Chronograf 1.7.10 (git: 2d67c8fa2d291644bee858da00ed13533344097a)

**Comment by endresjo** [comment]
Update: problem seems to be solved in Chronograf version 1.7.11. 
Installed this version as dcoker container (chronograf:1.7.11); everything is working as expected!
